sklearnex/preview/covariance/covariance.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+from scipy import sparse as sp
+from sklearn.covariance import EmpiricalCovariance as sklearn_EmpiricalCovariance
+from sklearn.utils import check_array
+
+from daal4py.sklearn._utils import control_n_jobs, run_with_n_jobs, sklearn_check_version
+from onedal.common.hyperparameters import get_hyperparameters
+from onedal.covariance import EmpiricalCovariance as onedal_EmpiricalCovariance
+
+from ..._device_offload import dispatch
+from ..._utils import PatchingConditionsChain, register_hyperparameters
+
+
+@register_hyperparameters({"fit": get_hyperparameters("covariance", "compute")})
+@control_n_jobs
```

Comment:
How we can test it?

Reply:
I don't get this changes. I would recommend to revert them and if it is required add it in seperate PR.
It is need good description and test cov for this as well.